### ISSUE TITLE
 Notify Push container remains on (Starting)

### ISSUE THREAD

**Issue by punkyard** [nextcloud/all-in-one#3750]
### Steps to reproduce
1. go to master container interface
2. update
3. restart containers

### Expected behavior 
All lights usely go green after containers have restarted

### Actual behavior 
Since AiO v7.5.x, Notify Push remains orange

### Host OS 
debian 11

#### Nextcloud AIO version 
27.1.3

#### Current channel 
7.6.2

#### Other valuable info 

<img width="583" alt="Capture d’écran 2023-11-14 à 18 59 37" src="https://github.com/nextcloud/all-in-one/assets/59349105/0159575e-e73a-4d57-98ce-6535f7044342">

**Comment by lachermeierm** [comment]
log file:

Waiting for Nextcloud to start...
/start.sh: line 51: /nextcloud/custom_apps/notify_push/bin/x86_64/notify_push: Permission denied

Nextcloud AIO v7.6.2

**Comment by szaimen** [comment]
Hi, this is now fixed with 7.7.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by marcatsmci** [comment]
```
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
[2023-11-16 17:43:02.671387 +00:00] ERROR [notify_push] src/main.rs:78: Self test failed: Error while communicating with nextcloud instance
[2023-11-16 17:43:07.903816 +00:00] WARN [notify_push::connection] src/connection.rs:94: Invalid credentials
```
But running ;-)

![image](https://github.com/nextcloud/all-in-one/assets/47881838/174611d9-aeab-46e9-94ba-201afdc6b770)
